README.md

```diff
@@ -35,10 +41,18 @@ sudo apt install yarp
 sudo apt remove yarp && sudo apt autoremove
 ```
 
-## Compilation
+## <a name="install_gz-sim-yarp-plugins"></a> Install gz-sim-yarp-plugins
 ~~~
 mkdir build
 cd build
 cmake ..
 make
+sudo make install
```

Comment:
In general it is not a good idea to install in `/usr/local` as it can quicky become messy and lead to unmantainable setups. I suggest instead to document:
```suggestion
mkdir build
cd build
cmake -DCMAKE_INSTALL_PREFIX=<desired_install_prefix> ..
make
make install
```

and then reference /path/to/the/install/folder/lib in the following docs.